src/app/docs/id/id-docs.component.html

```diff
@@ -100,8 +90,10 @@
       >
         <p>
           Don't use the ID directive when an element does not need to be referenced by its
-          ID attribute, since doing so adds unnecessary overhead to an application. The same rule
-          goes for assigning a hard-coded ID to an element, because it can lead to ID collisions.
+          ID attribute.
+        </p>
+        <p>
+          Don't use the ID directive to hard-code an ID to element, which can lead to ID collisions.
         </p>
       </sky-docs-design-guideline>
 
```

Comment:
In the demo, do we need all three examples? Looking at the code, it seems like the first and third examples are identical except for the `1` and `3`.

Reply:
It was really just to demonstrate the usage of `skyId` on multiple elements. I can remove the last one.